packages/components/src/components/hds/dropdown/list-item/interactive.hbs

```diff
@@ -30,7 +30,11 @@
         </span>
       {{/if}}
       <Hds::Text::Body class="hds-dropdown-list-item__interactive-text" @tag="span" @size="200" @weight="medium">
-        {{this.text}}
+        {{#if (has-block)}}
+          {{yield (hash Badge=(component "hds/badge" class="hds-dropdown-list-item__interactive-badge"))}}
```

Comment:
[suggestion] since we're yielding any content anyway, and we're not passing any specific argument to the badge (and the class name `hds-dropdown-list-item__interactive-badge` doesn't have a CSS style) what is the benefit (for the consumers, as well as for us) of yielding as contextual component the badge?

```suggestion
          {{yield}}
```

Reply:
That makes sense, and I considered that, but then I thought of the ApplicationState::Footer component. We specifically yield the intended children there, even though it doesn't provide any benefit. I thought it was more to inform the intention of the component API.

Reply:
I see your point 🤔. Maybe a difference (debatable) is that in the `ApplicationState::Footer` we know and want to have only these three yielded components (actions). Is the same here? (in my mind, I expect consumers to start yielding other things, but maybe we want to "guide" them in yielding only a badge. OK, resolving the conversation.

Reply:
That's a good question too. I'm going to talk to Lilith and Heather.